frontend/src/components/Pages/FrontPage/MissionOverview/StopMissionDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { Button, Dialog, Typography, Icon } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { TranslateText } from 'components/Contexts/LanguageContext'
+import { Icons } from 'utils/icons'
+import { useState, useEffect } from 'react'
+import { tokens } from '@equinor/eds-tokens'
+import { Mission } from 'models/Mission'
+import { useMissionControlContext } from 'components/Contexts/MissionControlContext'
+
+const StyledDisplayButtons = styled.div`
+    display: flex;
+    width: 410px;
+    flex-direction: columns;
+    justify-content: flex-end;
+    gap: 0.5rem;
+`
+
+const StyledDialog = styled(Dialog)`
+    display: grid;
+    width: 450px;
+`
+
+const StyledText = styled.div`
+    display: grid;
+    gird-template-rows: auto, auto;
+    gap: 1rem;
+`
+
+interface MissionProps {
+    mission: Mission
+}
+
+export enum ControlButton {
+    Pause,
+    Stop,
+    Resume,
+}
+
+export const StopMissionDialog = ({ mission }: MissionProps): JSX.Element => {
+    const [isStopMissionDialogOpen, setIsStopMissionDialogOpen] = useState<boolean>(false)
+    const [missionId, setMissionId] = useState<string>()
+    const { handleClick } = useMissionControlContext()
+
+    const handleDialog = ({ mission }: MissionProps) => {
+        setIsStopMissionDialogOpen(true)
+        setMissionId(mission.id)
+    }
+
+    useEffect(() => {
+        if (missionId != mission.id) setIsStopMissionDialogOpen(false)
+    }, [mission.id])
+
+    return (
+        <>
+            <Button variant="ghost_icon" onClick={() => handleDialog({ mission })}>
+                <Icon
+                    name={Icons.StopButton}
+                    style={{ color: tokens.colors.interactive.secondary__resting.rgba }}
+                    size={40}
+                />
+            </Button>
+
+            <StyledDialog open={isStopMissionDialogOpen} isDismissable={false}>
```

Comment:
Why don't we want it to be dismissable?

Reply:
I thought it is would be better to force the user to either cancel or stop, since we have the cancel button which provides the same functionality as dismissible. Do you think we should change it?

Reply:
We have generally used isDismissable and unless there is a reason to have it as false I would leave it as true for consistency.  If you prefer false that is fine true

Reply:
My reasoning was because in the tablet it can be easy to click somewhere and if you are trying to stop a mission and suddenly click somewhere and have to click the stop button again, that might not be ideal. Depends a bit how crucial are the situations in which the operator is stopping the mission and how urgent. But I'm probably not the best person to answer this question so I'll leave it up to you to decide!